utils/ort/src/main/kotlin/storage/HttpFileStorage.kt

```diff
@@ -35,6 +36,8 @@ import org.ossreviewtoolkit.utils.ort.OkHttpClientHelper
 import org.ossreviewtoolkit.utils.ort.execute
 
 private const val HTTP_CLIENT_CONNECT_TIMEOUT_IN_SECONDS = 30L
+private const val HTTP_CLIENT_KEEP_ALIVE_DURATION_IN_SECONDS = 5 * 60L
+private const val HTTP_CLIENT_MAX_IDLE_CONNECTIONS = 5
```

Comment:
It took me a while to understand that you're just repeating the default values here, in order to change to a non-default keep-alive duration only in the next commit. This could either be made explicit in the commit message, or (preferably) just squash the commits and say right away that you're customizing the keep-alive duration.

Speaking of the commit message, I find it also a bit confusing to refer to [1] twice in the same sentence.

Reply:
I've adjusted the commit message accordingly.